code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -172,6 +172,9 @@
 
 /obj/structure/closet/secure_closet/security/sec/PopulateContents()
 	..()
+	new /obj/item/gun/ballistic/automatic/pistol/enforcer

```

Comment:
You just casually added enforcers for all security officers.

Pls no.

Reply:
These aren't the same lockers as the regular ones that currently spawn